website/src/content/deployment-examples/index.mdx

```diff
@@ -53,11 +53,17 @@ This page outlines some of the common platforms and services Custom Applications
   >
     Learn more about deploying to Cloudflare Pages.
   </Card>
+  <Card
+    title="Google Cloud Storage"
```

Comment:
```suggestion
    title="Google Cloud with Cloud Storage"
```
as they named it [here](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#0)